packages/interactivity/docs/2-api-reference.md

```diff
@@ -363,20 +365,90 @@ store( "myPlugin", {
 </details>
 <br/>
 
-The `wp-on` directive is executed each time the associated event is triggered. 
+≈
```

Comment:
Is this intentional?

Reply:
No 🤦‍♂️

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/57931/commits/c8635c4132b548f97eb4136c39044881c8221867